obal/data/modules/srpm.py

```diff
@@ -0,0 +1,111 @@
+"""
+Build SRPM
+"""
+
+import shutil
+import os
+import subprocess
+from tempfile import mkdtemp
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.chdir import chdir
+
+def run_command(command):
+    output = subprocess.check_output(
+                ' '.join(command),
+                universal_newlines=True,
+                shell=True,
+                stderr=subprocess.STDOUT
+             )
+    return output
+
+
+def copy_sources(spec_file, package_dir, sources_dir, module):
+    command = ["spectool", "--list-files", spec_file]
+    sources = run_command(command)
+
+    for source in sources.split("\n"):
+        if len(source) == 0:
+            continue
+
+        source_link = source.split(' ')[1]
+
+        if not source_link.startswith('http'):
+            shutil.copy("%s/%s" % (package_dir, source_link), sources_dir)
+        else:
+            with chdir(package_dir):
+                run_command(["git-annex", "lock"])
+                annexed_files = run_command(["git-annex", "find", "--include='*'"])
```

Comment:
I think this results in double iteration. You iterate over all sources (in case of bundled NPM packages this may be 50) and now you're iterating over all annexed files which would result in copying every file 50 times, meaning 2500 copies.